templates/header.html

```diff
@@ -1,13 +1,14 @@
 {{ define "header" }}
 <head>
     <meta http-equiv="Content-Security-Policy" 
-        content="default-src 'self'; object-src 'none'; media-src 'none'; connect-src 'self'; font-src https://fonts.gstatic.com 'self'; child-src 'self'; style-src 'self'; img-src 'self' {{.LogoURL}}; script-src 'self'; form-action 'self';">
-    <link href="/assets/material-icons.css" rel="stylesheet">
-    <link rel="stylesheet" type="text/css" href="/assets/materialize-0.97.5.min.css">
-    <link rel="stylesheet" type="text/css" href="/assets/font-awesome.min-4.7.0.css">
-    <link rel="stylesheet" type="text/css" href="/assets/css.css">
-    <script type="text/javascript" src="/assets/jquery-3.5.1.min.js"></script>
-    <script type="text/javascript" src="/assets/materialize.min-0.97.5.js"></script>
-    <script type="text/javascript" src="/assets/script.js"></script>
+    content="default-src 'self'; object-src 'none'; media-src 'none'; connect-src 'self'; font-src https://fonts.gstatic.com 'self'; child-src 'self'; style-src 'self' 'sha384-Tn+eHgvLDlHfZ/Bd0HmrFRKnaocbJJECUsEAjjg2gey5liDBv1trMEyh2l7XC2C+' 'sha384-1ji7hb9jc+M2e4aPgCIK93lON9Hxx38Vo/3oNk9vrJsU8JbrdFdLs+VmVE1YNiuM' 'sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN' 'sha384-TeXUgdWM9e/rDL9CedCXsh4Z0a6aZJUiyD7Vfz3S+H1REIOyQEXhpneyhHZswE3a' 'nonce-{{.NonceCode}}'; img-src 'self' {{.LogoURL}}; script-src 'self' 'sha384-ZvpUoO/+PpLXR1lu4jmpXWu80pZlYUAfxl5NsBMWOEPSjUn/6Z/hRTt8+pR6L4N2' 'sha384-jnt1QI5LA9Z8CEqFV7YvpkT/kvVzzSDZbit0VjFaNiz/XtzoN8OA7z/RI/cbzs95' 'sha384-lCnwfMpb5VT9dtF36tAIDhczx0j9zjzzpFY7qIxdwU+0VPRYT7xtOTsVIVkcwbhm' 'nonce-{{.NonceCode}}'; form-action 'self'">
+    
+    <link href="/assets/material-icons.css" rel="stylesheet" integrity="sha384-Tn+eHgvLDlHfZ/Bd0HmrFRKnaocbJJECUsEAjjg2gey5liDBv1trMEyh2l7XC2C+">
+    <link rel="stylesheet" type="text/css" href="/assets/materialize-0.97.5.min.css" integrity="sha384-1ji7hb9jc+M2e4aPgCIK93lON9Hxx38Vo/3oNk9vrJsU8JbrdFdLs+VmVE1YNiuM">
+    <link rel="stylesheet" type="text/css" href="/assets/font-awesome.min-4.7.0.css" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN">
+    <link rel="stylesheet" type="text/css" href="/assets/css.css" integrity="sha384-TeXUgdWM9e/rDL9CedCXsh4Z0a6aZJUiyD7Vfz3S+H1REIOyQEXhpneyhHZswE3a">
+    <script type="text/javascript" src="/assets/jquery-3.5.1.min.js" integrity="sha384-ZvpUoO/+PpLXR1lu4jmpXWu80pZlYUAfxl5NsBMWOEPSjUn/6Z/hRTt8+pR6L4N2"></script>
+    <script type="text/javascript" src="/assets/materialize.min-0.97.5.js" integrity="sha384-jnt1QI5LA9Z8CEqFV7YvpkT/kvVzzSDZbit0VjFaNiz/XtzoN8OA7z/RI/cbzs95"></script>
+    <script type="text/javascript" src="/assets/script.js" integrity="sha384-lCnwfMpb5VT9dtF36tAIDhczx0j9zjzzpFY7qIxdwU+0VPRYT7xtOTsVIVkcwbhm"></script>
```

Comment:
I like the idea of having these integrity checksums! I'm not too familiar with that, is there a way these could be generated automatically, for example during the build or during the stage when the assets are embedded into the binary? That would make further updates to the assets easier.

Reply:
I am not sure, but it should be do-able.  Mobile browsers complain quit a bit without those is the reason they were added.

Reply:
I think that will be difficult sadly due to the use of pkger to embed the static content.  I didn't notice you were doing that before, so I have removed the hashes, moving the scripts to be in the templates/assets folder has fixed the mobile problem.

Reply:
Okay I have added a script for the integrity checksums.  And mobile safari seems happy.  I'm not sure how you want to integrate the script into the build pipeline